docs/supported_apis/dataframe_supported.rst

```diff
@@ -35,7 +41,7 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``any``                    | `any`_                    | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``append``                 | `append`_                 | Y                      |                                                    |
+| ``append``                 | `append`_                 | Y                      | **Omnisci**:``Y``?                                 |
```

Comment:
It should be supported why a question tag? I get plenty of warning when running `append` but not a default-to-pandas one 😄 (except for transpose)
```python
>>> import modin.pandas as pd
>>> df = pd.DataFrame({"a": [1, 2, 2, 1], "b": [3, 4, 5, 6]})
>>> df.append(pd.Series([10, 20], index=["a", "b"]), ignore_index=True)
UserWarning: Distributing <class 'list'> object. This may take some time.
FutureWarning: pandas.Int64Index is deprecated and will be removed from pandas in a future version. Use pandas.Index with the appropriate dtype instead.
UserWarning: Due to https://github.com/pandas-dev/pandas/issues/35092, Pandas ignores sort=False; Modin correctly does not sort.
UserWarning: <function DataFrame.transpose> defaulting to pandas implementation.
Please refer to https://modin.readthedocs.io/en/stable/supported_apis/defaulting_to_pandas.html for explanation.
FutureWarning: pandas.Int64Index is deprecated and will be removed from pandas in a future version. Use pandas.Index with the appropriate dtype instead.
    a   b
0   1   3
1   2   4
2   2   5
3   1   6
4  10  20
```

Reply:
There were doubts whether all params supported

Reply:
eg there is a ``sort`` param

Reply:
`sort` and `ignore_index` parameters are actually ignored by [`OmnisciOnNativeDataframe.concat`](https://github.com/modin-project/modin/blob/95d71b2bde0ceb40cb1b64fa762b3009a94168dd/modin/experimental/core/execution/native/implementations/omnisci_on_native/dataframe/dataframe.py#L1125) in several cases